docs/CHANGELOG.rst

```diff
@@ -25,6 +25,11 @@ Changed
 -------
 - **BACKWARD INCOMPATIBLE:** Require Resolwe 42.x
 
+Fixed
+-----
+- Normalize project name from `resolwe-bio` to `resolwe_bio` in about file
```

Comment:
When rebasing, make sure to properly place this changelog entry to unreleased section.

Reply:
Fixed